### ISSUE TITLE
[Question] Full semantic annotations

### ISSUE THREAD

**Issue by alfieroddan** [IMSY-DKFZ/htc#24]
## :question: Question

Is it possible to access the semantic annotations?

### Description

Hi there just a quick question about the annotations.

From what I gather the full dataset we are able to download from [the website](https://heiporspectral.org) only contains binary segmentation annotations (not semantic) like the below:

![Screenshot 2024-01-09 at 17 50 01](https://github.com/IMSY-DKFZ/htc/assets/51797647/371ad450-e1cf-4f97-ae35-8189e2bc4c4b)

As opposed to the semantic annotations see in Figure 8 of [Robust deep learning-based semantic organ segmentation in Hyperspectral images](https://doi.org/10.1016/j.media.2022.102488).

Currently if we train using the full dataset and your framework the model is learning some organs as a background class and also its own class. Is this the intended optimisation or is the full annotation not released?

Once again thank you very much for the work on this package, I have been enjoying getting to grips to it and feel as if I'm almost at the point of contribution.

**Comment by JanSellner** [comment]
You are right, the HeiPorSPECTRAL dataset only contains polygon annotations with (usually) one organ in an image.

In our MIA paper, we used fully semantically annotated images which are not open source and the corresponding images are also from different subjects (no overlap with the HeiPorSPECTRAL dataset).

Are you using one of our pretrained models and then finetune on the HeiPorSPECTRAL dataset? The classes do not overlap completely, e.g., there is no background class in the HeiPorSPECTRAL dataset, so some confusions may be expected. During finetuning, I could also imagine that the network forgets some classes (like background) if it does not see them anymore. Early stopping may be helpful here. Alternatively, you could also first make a prediction on all images from the HeiPorSPECTRAL dataset using our pretrained image model and use the predictions as new semantic labels (with the actual labels from the HeiPorSPECTRAL superimposed of course).

Thanks again for the interest in our work and I hope that our models and the dataset are still useful for you even if the semantic dataset isn't open source.

**Comment by hafsa390** [comment]
Hi @JanSellner , will the fully annotated multi-organ dataset be publicly available?

**Comment by JanSellner** [comment]
@hafsa390 It might be in the future but at the moment there are no concrete plans.

**Comment by JanSellner** [comment]
The fully annotated multi-organ images are now publicly available through [xeno-spectral](https://spectralverse-heidelberg.org/xeno-spectral/)!
